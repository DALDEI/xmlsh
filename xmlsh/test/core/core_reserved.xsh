words="catch for if elif then else fi do case while until try esac finally return done in function local"
for w in $words ; do echo $w ; done

echo $words
echo catch for if elif then else fi do case while until try esac finally return done in function local
echo catch= for= catch, fi, if, =if if= :if if: 
echo if ,if if, ,if, do ,,do =do =:do done ,done done{} {done}