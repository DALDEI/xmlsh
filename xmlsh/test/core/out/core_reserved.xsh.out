catch
for
if
elif
then
else
fi
do
case
while
until
try
esac
finally
return
done
in
function
local
catch for if elif then else fi do case while until try esac finally return done in function local
catch for if elif then else fi do case while until try esac finally return done in function local
catch= for= catch , fi , if , =if if= :if if:
